uber/site_sections/api.py

```diff
@@ -94,3 +94,33 @@ def revoke_api_token(self, session, id=None):
         api_token.revoked_time = datetime.now(pytz.UTC)
         raise HTTPRedirect(
             'index?message={}', 'Successfully revoked API token')
+
+    @public
+    @staticmethod
+    def stripe_webhook_handler(request=None):
+        if not request or not request.body:
+            return "Request required"
+        sig_header = request.META['HTTP_STRIPE_SIGNATURE']
```

Comment:
This is copy-pasted from Stripe's documentation here: https://stripe.com/docs/payments/handling-payment-events#build-your-own-webhook

I don't know enough about request headers to know if they're wrong -- if you think they are, I'll go ahead and make that change. :)

Reply:
Ah, yeah from looking at those docs, ``request.META`` is a Django thing.  Since we're using CherryPy, it should definitely be ``cherrypy.request.headers``.

Reply:
Ooh, great catch! Thank you!